src/main/java/com/imsweb/datagenerator/naaccr/NaaccrDataGenerator.java

```diff
@@ -126,6 +129,9 @@ public NaaccrDataGenerator(boolean useMaidenNameField) {
         _tumorRules.add(new StagingInputRule());
         _tumorRules.add(new FacilityRule());
         _tumorRules.add(new PhysicianRule());
+        _tumorRules.add(new TumorTextRule());
```

Comment:
The list of fields set by the rules is listed in the README.md file in the top folder of the project. Can you please add three entries for the three new rules in that section? Thanks.